.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+    env: {
+        es6: true,
+        browser: true,
+        node: true,
+    },
+    extends: ['airbnb/base', 'prettier'],
+    parserOptions: {
+        sourceType: 'module',
+        ecmaVersion: 'latest',
+    },
+    rules: {
+        'linebreak-style': ['error', 'unix'],
+        camelcase: 0,
+        'class-methods-use-this': 0,
+        'consistent-return': 0,
```

Comment:
IMO this rule is useful since it makes method calls more predictable

Reply:
Agreed